cmd/yaman/run.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/docker/docker/pkg/namesgenerator"
+	"github.com/spf13/cobra"
+	"github.com/willdurand/containers/cmd/yaman/container"
+	"github.com/willdurand/containers/cmd/yaman/shim"
+	"github.com/willdurand/containers/cmd/yaman/yaman"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:          "run <image> [<command> [<args>...]]",
+		Short:        "Run a command in a new container",
+		RunE:         run,
+		SilenceUsage: true,
+		Args:         cobra.MinimumNArgs(1),
+	}
+	cmd.Flags().BoolP("detach", "d", false, "run container in background and print container ID")
+	cmd.Flags().String("name", "", "assign a name to the container")
+	cmd.Flags().String("runtime", "", "runtime to use for this container")
+	rootCmd.AddCommand(cmd)
+}
+
+func run(cmd *cobra.Command, args []string) error {
+	rootDir, _ := cmd.Flags().GetString("root")
+	debug, _ := cmd.Flags().GetBool("debug")
+
+	detach, _ := cmd.Flags().GetBool("detach")
```

Comment:
I think we could have `--rm` pretty easily too, just need to pass that to the container options and on cleanup() we would call Delete().

Reply:
done